test/integration/associations/belongs-to-many.test.js

```diff
@@ -11,6 +11,17 @@ const chai = require('chai'),
   current = Support.sequelize,
   dialect = Support.getTestDialect();
 
+// Db2 returns UUID as buffer and expect fails to compare buffer
+// This function takes care of this comparision issue.
+/*
```

Comment:
Why is this commented out?

Reply:
This is nothing. Just some unused code. removed it.